### DISCUSSION TITLE
关于 Taro 的 Minor 版本代号的命名类别投票

### DISCUSSION THREAD

**Discussion by Chen-jj** [NervJS/taro#12680] [Polls]
背景：从 Taro `v3.6` 开始，对该 Minor 或 Major 版本作出较大贡献的同学，将拥有对该版本的英文代号的**提名权**。最终经 Taro 技术委员会和合作者共同在提名中确认最终的版本号。希望能对贡献者起到激励作用。

因此，本投票是为了决定未来 Taro 版本代号的命名需要限制在哪一个相对具体的类别中，如：动物、植物等。所有开发者均有权投票，最终的类别将由技术委员会和合作者共同选出并进行公示。

目前对 Taro v3.6 版本拥有代号命名投票权的同学：

- @xueshuai - 升级 PostCSS8
- @AdvancedCat - 小程序实现 History API
- @bigmeow - 优化重构 Mini CI 插件
- @robinv8  - 实现类型系统的自动化

**Comment by AdvancedCat** [comment]
是否可以考虑将「游戏/动漫」纳入类别中，开发者(特别是Z世代)对这个类别可能更感兴趣一些

**Comment by zhiqingchen** [comment]
是否可以考虑将「天文」纳入类别中，比较有意思

**Comment by honlyHuang** [comment]
电影名

**Comment by Chen-jj** [comment]
人文类感觉容易产生争议或存在合规问题

**Comment by ZakaryCode** [comment]
不能多选么😂

**Comment by innocces** [comment]
_(:з」∠)_  fate 党感觉是不是可以用英灵的. 这种很cool. 也不会有什么争议

**Comment by honlyHuang** [comment]
地域美食，这样命名者可以给出自己喜爱的美食，也可以给出家乡美食，或者推广小众美食，都很棒；
大众的：热干面、螺蛳粉、顺德鱼生
小众的：生腌（潮汕毒药）、烟薯 25 号、六也鱼怪
